formulaire/src/main/java/fr/openent/formulaire/service/impl/DefaultQuestionChoiceService.java

```diff
@@ -1,20 +1,27 @@
 package fr.openent.formulaire.service.impl;
 
 import fr.openent.form.core.enums.I18nKeys;
+import fr.openent.form.core.models.QuestionChoice;
+import fr.openent.form.helpers.FutureHelper;
 import fr.openent.form.helpers.I18nHelper;
 import fr.openent.formulaire.service.QuestionChoiceService;
 import fr.wseduc.webutils.Either;
+import io.vertx.core.Future;
 import io.vertx.core.Handler;
+import io.vertx.core.Promise;
+import io.vertx.core.json.Json;
 import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
 import org.entcore.common.sql.Sql;
 import org.entcore.common.sql.SqlResult;
 
 import static fr.openent.form.core.constants.Fields.*;
-import static fr.openent.form.core.constants.Tables.QUESTION_CHOICE_TABLE;
-import static fr.openent.form.core.constants.Tables.SECTION_TABLE;
+import static fr.openent.form.core.constants.Tables.*;
 
 public class DefaultQuestionChoiceService implements QuestionChoiceService {
+    private static final Logger log = LoggerFactory.getLogger(QuestionChoiceService.class);
```

Comment:
utilise le nom de ta classe d'implémentation direct

```suggestion
    private static final Logger log = LoggerFactory.getLogger(DefaultQuestionChoiceService.class);
```